nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractExecuteSQL.java

```diff
@@ -454,17 +469,14 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             //  pass the original flow file down the line to trigger downstream processors
             if (fileToProcess == null) {
                 // This can happen if any exceptions occur while setting up the connection, statement, etc.
-                logger.error("Unable to execute SQL select query {} due to {}. No FlowFile to route to failure",
-                        new Object[]{selectQuery, e});
+                logger.error("Unable to execute SQL select query {} due to {}. No FlowFile to route to failure", selectQuery, e);
```

Comment:
This is a good opportunity to correct the unnecessary placeholder use of `due to {}`. The exception will be logged in a stack trace, so it should not be included as a placeholder value.
```suggestion
                logger.error("Unable to execute SQL select query [{}] No FlowFile to route to failure", selectQuery, e);
```